src/components/ToDoItem/ToDoItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const ToDoItem: React.FC<Props> = ({ todo }) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+    >
+      {/* eslint-disable jsx-a11y/label-has-associated-control */}
```

Comment:
[Not critical]
Disabling eslint rules is not a big deal for a homework. But for a real work, I would not recommend to disable eslint rules if errors are produced on your code. These rules were most likely added for some reason, so it's better to fix your code to align it with rules. If this is not possible or you feel that a rule doesn't give enough profit, you can discuss changing its configuration or even removal with the team.